tuf/store/filestore.go

```diff
@@ -53,7 +55,14 @@ func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 		}
 		return nil, err
 	}
-	return meta, nil
+	if size == -1 {
```

Comment:
I'm open to changing this to 0 if there's a consensus, though for me 0 feels more like a `nil` than "unlimited".  I agree that we should be consistent across notary with whatever we choose.

I chose a negative value here to be consistent with how Go handles `strings.Replace` -- a negative value replaces every occurrence in your input string.